external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -170,10 +182,13 @@ private void initialize(Collection<TopicPartition> partitions) {
                 final long fetchOffset = doSeek(tp, committedOffset);
                 setAcked(tp, fetchOffset);
             }
-            initialized = true;
+
+            task.start();
```

Comment:
I believe @srdo says is true. Once a rebalance happens, I don't think commits can be done to kafka. The purpose of the onPartitionsRevoked callback is to handle persisting offsets in a separate storage.